src/openloco/graphics/image_ids.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#define UNUSED_IMG(x) (x)
+
+namespace openloco::image_ids
+{
+    constexpr uint32_t close_button = 2321;
+
+    constexpr uint32_t chris_sawyer_logo = 3616;
```

Comment:
These are the small variants of the logos.

![image](https://user-images.githubusercontent.com/737603/41914573-d433ebaa-7953-11e8-95c4-d4ad6cf8660e.png)

Reply:
Right, the ones used in the intro sequence...